.github/workflows/build_hub.yml

```diff
@@ -68,9 +68,14 @@ jobs:
 
       - name: Build Prime Router Package
         run: bash ./build.sh -- gradle clean package -x fatjar -Pshowtests
+        
       - name: Publish Unit Test Results
         uses: EnricoMi/publish-unit-test-result-action@v1
-        if: always()
+        # Per https://github.com/EnricoMi/publish-unit-test-result-action#support-fork-repositories-and-dependabot-branches
+        if: >
+          always() &&
```

Comment:
remove the `always()` since it's used in `&&` and it always returns `true` anyway
see https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#always

Not quite sure why the referenced link instructs you to include the `always()` call (which it does)

Reply:
I agree the always is needed, but I do want to keep the code as references in the instructions.  The always does not hurt.

Reply:
is _not_ needed, you mean?

Reply:
In Slack combo we agreed that it was ok to leave the always.